serverstats/serverstats.py

```diff
@@ -1466,5 +1466,7 @@ def joined(member: discord.Member):
             em.set_footer(text="Page {} of {}".format(count, len(x)))
             count += 1
             emoji_embeds.append(em)
-
-        await menu(ctx, emoji_embeds, DEFAULT_CONTROLS)
+        try:
+            await menu(ctx, emoji_embeds, DEFAULT_CONTROLS)
+        except IndexError:
```

Comment:
This can be differently handled instead of using a try except at the end of this command. For example you can check the len of guild emojis before create the embed, and if the result is less than 1 return your message that say there's no emojis in the selected server.